### PR TITLE
Add MetCoOp L90 defintion and some updated doc

### PR DESCRIPTION
**Pull request by ewhelan**
Add the MetCoOp L90 defintion for use by jbdiagnose

### CODE REVIEW

File: doc/src/jbdiagnose.md [hunk 1]
```diff
@@ -45,7 +45,19 @@ plotjbdiag -l 50 -t vercor -p QQ -r ./ -e IRELAND75L65
...
+The first part of the diagnostic tool is a simple standalone fortran program (jbdiagnose.F90) that reads the two main background error statistics files (jb\_filename.bal and jb\_filename.cv), calculates various diagnostic quantities and writes these diagnostic quantitiesin simple ASCII files that are then used by _plotjbdiag_ to produce various graphs. The input background error statistics files are opened with the local file names _stabal96.bal_ and _stabal96.cv_ and a namelist variable _gsizein_, describing the grid resolution, and namelist variables describing the vertical levels (_ahalf\_nl_ and _bhalf\_nl_, ) are needed.
+
+The following vertical level definitions are available for use:
+| Name       | Description                      |
+| ---------- | -------------------------------- |
+|hirlL60.def |HIRLAM 60 level definition        |
```
Comment (roelstappers) [inline doc/src/jbdiagnose.md]:
```suggestion
The following vertical level definitions are available for use:

| Name       | Description                      |
| :--        | :--                              |
|`hirlL60.def` |HIRLAM 60 level definition        | 
```

Comment (roelstappers) [reply doc/src/jbdiagnose.md]: The empty line before the markdown table is needed or else you wil get: 

Screenshot from 2024-06-28 15-06-04

File: doc/src/jbdiagnose.md [hunk 2]
```diff
@@ -45,7 +45,19 @@ plotjbdiag -l 50 -t vercor -p QQ -r ./ -e IRELAND75L65
...
+The first part of the diagnostic tool is a simple standalone fortran program (jbdiagnose.F90) that reads the two main background error statistics files (jb\_filename.bal and jb\_filename.cv), calculates various diagnostic quantities and writes these diagnostic quantitiesin simple ASCII files that are then used by _plotjbdiag_ to produce various graphs. The input background error statistics files are opened with the local file names _stabal96.bal_ and _stabal96.cv_ and a namelist variable _gsizein_, describing the grid resolution, and namelist variables describing the vertical levels (_ahalf\_nl_ and _bhalf\_nl_, ) are needed.
```
Comment (roelstappers) [inline doc/src/jbdiagnose.md]:
```suggestion
The first part of the diagnostic tool is a simple standalone fortran program (`jbdiagnose.F90`) that reads the two main background error statistics files (`jb_filename.bal` and `jb_filename.cv`), calculates various diagnostic quantities and writes these diagnostic quantitiesin simple ASCII files that are then used by `plotjbdiag` to produce various graphs. The input background error statistics files are opened with the local file names `stabal96.bal` and `stabal96.cv` and a namelist variable `gsizein`, describing the grid resolution, and namelist variables describing the vertical levels (`ahalf_nl` and `bhalf_nl`, ) are needed.
```

File: doc/src/jbdiagnose.md [hunk 3]
```diff
@@ -45,7 +45,19 @@ plotjbdiag -l 50 -t vercor -p QQ -r ./ -e IRELAND75L65
...
+| Name       | Description                      |
+| ---------- | -------------------------------- |
+|hirlL60.def |HIRLAM 60 level definition        | 
+|harmL65.def |Harmonie 65 level definition      | 
+|mcL90.def   |MetCoOp 90 level definition       | 
+|mfL70.def   |Meteo France 70 level definition  | 
+|mfL90.def   |Meteo France 90 level definition  | 
+
+
+ Various information about the domain geometry is extracted from the input files. One example of a script to run the diagnostic tool is provided in jbdiagnose.sh. The output ASCII files are stored in the
```
Comment (roelstappers) [inline doc/src/jbdiagnose.md]:
```suggestion
|`harmL65.def` |Harmonie 65 level definition      | 
|`mcL90.def`   |MetCoOp 90 level definition       | 
|`mfL70.def`   |Meteo France 70 level definition  | 
|`mfL90.def`   |Meteo France 90 level definition  | 


 Various information about the domain geometry is extracted from the input files. One example of a script to run the diagnostic tool is provided in `jbdiagnose.sh`. The output ASCII files are stored in the
```

**Comment by ewhelan** [comment]
@roelstappers : You're keeping your eyes 👀 on everything!

**Pull request merged**